cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -853,3 +903,112 @@ select {
     width: 100%;
   }
 }
+
+.xblock-render .add-xblock-component {
+  background: transparent;
+  padding: $baseline;
+
+  .new-component {
+    h5 {
+      margin-bottom: ($baseline * 1.2);
+      font-size: 22px;
+      font-weight: 700;
+      color: $black;
+    }
+
+    .new-component-type {
+      display: flex;
+      flex-wrap: wrap;
+      gap: ($baseline * .6);
+      align-items: center;
+      justify-content: center;
+
+      .add-xblock-component-button {
+        box-shadow: 0 1px 2px rgba(0, 0, 0, .15), 0 1px 4px rgba(0, 0, 0, .15);
+        width: 17.6rem;
+        height: 11rem;
+        color: $primary;
+        border-color: $primary;
+        background: transparent;
+        margin: 0;
+        display: inline-flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        gap: ($baseline * .4);
+
+        &:hover {
+          color: darken($primary, 10%);
+          background-color: lighten($primary, 80%);
+          border-color: darken($primary, 15%);
+        }
+
+        .large-template-icon {
+          width: 24px;
+          height: 24px;
+          background: $primary;
+
+          @each $name, $file in $template-icon-map {
+            &.large-#{$name}-icon {
+              mask: url("#{$static-path}/images/#{$file}.svg") center no-repeat;
+            }
+          }
+        }
+
+        .name {
+          color: inherit;
+          font-size: 1.575rem;
+          font-weight: 400;
+        }
+
+        .beta {
+          color: $white;
+          background-color: $primary;
+          padding: 0.2rem 0.8rem 0.4rem;
+          font-size: 1.35rem;
+          font-weight: 700;
+          line-height: 1;
+          margin: -0.6rem 0 0;
+        }
+      }
+    }
+  }
+}
+
+.paste-component {
+  margin: 2.4rem 1rem 0;
+
+  .paste-component-whats-in-clipboard .clipboard-details-popup {
+    right: -1rem;
+  }
+
+  .paste-component-button.button {
```

Comment:
[question]: Can we try using [mixins](https://github.com/openedx/edx-platform/blob/master/cms/static/sass/elements/_course-unit-mfe-iframe.scss) for the new styles in this file?

Reply:
Moved to mixins

Reply:
Resolved